### PR TITLE
Dodrio non-fatal vsc_home/data

### PR DESCRIPTION
**Pull request by stdweird**

### CODE REVIEW

File: files/checkpaths.sh [hunk 1]
```diff
@@ -93,7 +93,19 @@ function errormsg () {
     else
         # index -1 etc are supported, so make sure the index is  > 0
         if [ "$1" -ge $ECSTART ]; then
-            echo "${NAMES[$1 - $ECSTART]}"
+            name="VSC_${NAMES[$1 - $ECSTART]}"
+            if [ "$name" == 'VSC_INSTITUTE_LOCAL' ]; then
+                name = "APPS"
+            elif [ "$name" == "VSC_SCRATCH" ]; then
+                if [ "$CHECKPATHS_CLUSTER" == 'dodrio' ]; then
+                    name = "user homedir HOME / $name"
```
Comment (wdpypere) [inline files/checkpaths.sh:101]: no spaces for variable assignment in bash

File: files/checkpaths.sh [hunk 2]
```diff
@@ -93,7 +93,19 @@ function errormsg () {
     else
         # index -1 etc are supported, so make sure the index is  > 0
         if [ "$1" -ge $ECSTART ]; then
-            echo "${NAMES[$1 - $ECSTART]}"
+            name="VSC_${NAMES[$1 - $ECSTART]}"
+            if [ "$name" == 'VSC_INSTITUTE_LOCAL' ]; then
+                name = "APPS"
```
Comment (wdpypere) [inline files/checkpaths.sh:98]: idem

File: files/checkpaths.sh [hunk 3]
```diff
@@ -93,7 +93,19 @@ function errormsg () {
...
+            elif [ "$name" == "VSC_SCRATCH" ]; then
+                if [ "$CHECKPATHS_CLUSTER" == 'dodrio' ]; then
+                    name = "user homedir HOME / $name"
+                fi
+            elif [ "$name" == "VSC_HOME" ]; then
+                 if [ "$CHECKPATHS_CLUSTER" != 'dodrio' ]; then
+                    name = "user homedir $name"
```
Comment (wdpypere) [inline files/checkpaths.sh:105]: idem

File: files/checkpaths.sh [hunk 4]
```diff
@@ -93,7 +93,19 @@ function errormsg () {
     else
         # index -1 etc are supported, so make sure the index is  > 0
         if [ "$1" -ge $ECSTART ]; then
-            echo "${NAMES[$1 - $ECSTART]}"
+            name="VSC_${NAMES[$1 - $ECSTART]}"
+            if [ "$name" == 'VSC_INSTITUTE_LOCAL' ]; then
```
Comment (itkovian) [inline files/checkpaths.sh:97]:
```suggestion
            if [ "$name" == "VSC_INSTITUTE_LOCAL" ]; then
```

**Pull request merged**